lib/libutee/include/tee_internal_api_extensions.h

```diff
@@ -55,4 +55,11 @@ void *tee_map_zi(size_t len, uint32_t flags);
  */
 TEE_Result tee_unmap(void *buf, size_t len);
 
+/*
+ * Convert a UUID string @s into a TEE_UUID @uuid
+ * Expected format for @s is: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+ * 'x' being any hexadecimal digit (0-9a-fA-F)
+ */
+TEE_Result tee_parse_uuid(const char *s, TEE_UUID *uuid);
```

Comment:
How about `tee_uuid_from_str()` instead?

Reply:
+1

Reply:
Yup I like it better too